app/javascript/tests/history/History.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button } from 'react-bootstrap';
+import _ from 'lodash';
+import History from '../../components/history/History.js'
+import { mockHistory1, mockHistory2 } from '../mocks/mockHistories'
+
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+
+beforeEach(() => {
+  window.BASE_PATH = ""
+})
+
+describe('History', () => {
+  it('Properly renders non-comment histories', () => {
+    const wrapper = shallow(<History key={mockHistory1.id} history={mockHistory1} authenticity_token={authyToken} />);
+    const divEditId = "edit-history-item-" + mockHistory1.id;
+    const divDeleteId = "delete-history-item-" + mockHistory1.id;
+    expect(wrapper.find('b').exists()).toBeTruthy();
+    expect(wrapper.find('b').text()).toEqual(mockHistory1.created_by);
```

Comment:
Please add checks for the relative date and timestamp, as well as the history text contained in the card body